src/mappings.go

```diff
@@ -121,18 +128,33 @@ func convertMappingsJSONToMappings(json MappingsJSON) (Mappings, error) {
 		if k == "" {
 			return nil, fmt.Errorf("Empty key cannot be included.  Note: Corresponding value is '%s'", v)
 		}
-		if v == "" {
-			// Note: Ignore if dist is specified 'null' in JSON
-			continue
-		}
-		if v[0] != '~' && v[0] != '/' {
-			return nil, fmt.Errorf("Value of mappings must be an absolute path like '/foo/.bar' or '~/.foo': %s", v)
-		}
-		p, err := abspath.ExpandFromSlash(v)
-		if err != nil {
-			return nil, err
+		switch v := v.(type) {
+		case string:
+			if v == "" {
+				// Note: Ignore if dist is specified 'null' in JSON
+				continue
+			}
+			p, err := expandPath(v)
+			if err != nil {
+				return nil, err
+			}
+			m[k] = []abspath.AbsPath{p}
+		case []interface{}:
+			m[k] = make([]abspath.AbsPath, 0, len(v))
+			for _, iface := range v {
+				s, ok := iface.(string)
+				if !ok {
+					return nil, fmt.Errorf("Type of value must be string or string[]: %v", v)
+				}
```

Comment:
`expandPath` will crash when the value is empty string. Please check `s` is empty or net before calling it like above `case string:` clause.

Reply:
Fixed.
https://github.com/rhysd/dotfiles/pull/7/files#diff-4954e3d3c579a6d641c7c1f1c344eb86R144-R146